src/MOI_wrapper.jl

```diff
@@ -274,7 +284,7 @@ function MOI.copy_to(dest::Optimizer, src::MOI.ModelLike)
         MOI.PositiveSemidefiniteConeTriangle,
     )
     vis_src = MOI.get(src, MOI.ListOfVariableIndices())
-    if length(vis_src) < length(index_map.var_map)
+    if length(vis_src) != length(index_map.var_map)
```

Comment:
Ooops. I guess this should be a separate PR. Is the current `<` correct? Shouldn't it be the other way around (or `!=`)?